packages/core/theme-module/theme/components/AppHeader.vue

```diff
@@ -3,40 +3,49 @@
     active-sidebar="activeSidebar"
     @click:cart="toggleCartSidebar"
     @click:account="onAccountClicked"
+    :cartItemsQty="cartTotalItems"
     >
     <template #logo>
       <nuxt-link to="/" class="sf-header__logo">
         <SfImage src="/icons/logo.svg" alt="Vue Storefront Next" class="sf-header__logo-image"/>
       </nuxt-link>
     </template>
     <template #navigation>
-      <nuxt-link to="/c/women">
-        <SfHeaderNavigationItem>
+      <SfHeaderNavigationItem>
+        <nuxt-link to="/c/women">
           WOMEN
-        </SfHeaderNavigationItem>
-      </nuxt-link>
-      <nuxt-link to="/c/men">
-        <SfHeaderNavigationItem>
+        </nuxt-link>
+      </SfHeaderNavigationItem>
+      <SfHeaderNavigationItem>
+        <nuxt-link to="/c/men">
           MEN
-        </SfHeaderNavigationItem>
-      </nuxt-link>
-      <nuxt-link to="/c/cat">
-        <SfHeaderNavigationItem>
+        </nuxt-link>
+      </SfHeaderNavigationItem>
+      <SfHeaderNavigationItem>
+        <nuxt-link to="/c/cat">
           KIDS
-        </SfHeaderNavigationItem>
-      </nuxt-link>
+        </nuxt-link>
+      </SfHeaderNavigationItem>
     </template>
   </SfHeader>
 </template>
 
 <script>
 import { SfHeader, SfImage } from '@storefront-ui/vue';
 import uiState from '~/assets/ui-state';
+import { useCart } from '<%= options.composables %>';
+import { getCartTotalItems } from '<%= options.helpers %>';
+import { computed } from '@vue/composition-api';
 const { toggleCartSidebar, toggleLoginModal } = uiState;
 
 export default {
   setup() {
+    const { cart } = useCart();
+    const cartTotalItems = computed(() => {
+      return getCartTotalItems(cart.value).toString();
```

Comment:
Why this toString is needed?

Reply:
The `cartItemsQty` prop on `SfHeader` is declared as `String` so it pollutes the console with warnings when we put number there. 
Probably it should be reported to UI team, quantity is always a number...